elm.js

```diff
@@ -0,0 +1,6099 @@
+(function(scope){
```

Comment:
@SimonLab do we _want_ to be adding the (_compiled_) `elm.js` file to GitHub? 
is it not redundant if people are using `elm-live` to run it?

Reply:
At the moment unfortunately I need this file at the moment.
I'm using it on Heroku and I haven't find a way to create this file on compilation with the Heroku static buildpack yet.

Reply:
oh ... 💭 
The way we did it on OA was to compile it on CI and upload the compiled file to S3. 💡 
But that's overkill for this MVP (_for now_).
OK. let's just merge the compiled file for now.

Reply:
I'm using the same method to deploy Elm as described here: https://github.com/dwyl/elm-pwa-example/issues/3#issuecomment-571941816